Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1310,13 +1308,16 @@ class BrowserViewController: UIViewController {
                 // There are cases in which the page will still show a loading animation or nothing when the screenshot is being taken,
                 // depending on internet connection
                 // Issue created: https://github.com/mozilla-mobile/firefox-ios/issues/7003
-                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(1000)) {
+                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(500)) {
```

Comment:
Nitpick: We could define a constant with an explicit name so `.milliseconds(500)` is replaced by that constant here and on line 1318?